lib/network_manager.go

```diff
@@ -214,16 +238,79 @@ func (nm *NetworkManager) startRemoteNodeCleanup() {
 // ## Handlers (Peer, DeSoMessage)
 // ###########################
 
-// _handleVersionMessage is called when a new version message is received. It is a wrapper around the handshake's
-// handleVersionMessage function.
+// _handleVersionMessage is called when a new version message is received.
 func (nm *NetworkManager) _handleVersionMessage(origin *Peer, desoMsg DeSoMessage) {
-	nm.handshake.handleVersionMessage(origin, desoMsg)
+	if desoMsg.GetMsgType() != MsgTypeVersion {
```

Comment:
Ok cool I see this is what you did with the handshake manager. I love this consolidation.